bookdown/before_body.tex

```diff
@@ -0,0 +1,38 @@
+\begin{titlepage}
```

Comment:
KOMA supports title pages. Why don't we use that?

Reply:
I did not know that and had the snippet from my MA thesis still available.